sale_stock_picking_blocking/views/res_partner_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!--Copyright 2019 Eficent Business and IT Consulting Services S.L.-->
+  <!--(http://www.eficent.com)-->
+<!--License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).-->
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <group name="sale" position="inside">
+                    <field name="default_delivery_block"
+                        attrs="{'invisible': ['|', ('is_company', '=', False), ('parent_id', '!=', False)]}"/>
```

Comment:
I think **Default Delivery Block Reason** field should be visible for Individual Customer.

Reply:
@sudhir-erpharbor please fix if this makes sense, so this PR can be merged.

> I think **Default Delivery Block Reason** field should be visible for Individual Customer.